projects/cryptography/project.yaml

```diff
@@ -0,0 +1,12 @@
+fuzzing_engines:
+- libfuzzer
+homepage: https://github.com/pyca/cryptography
+language: python
+main_repo: https://github.com/pyca/cryptography
+sanitizers:
+- address
+primary_contact: "alex.gaynor@gmail.com"
```

Comment:
@alex FYI you're set as the primary contact -- is that okay?

Reply:
That's fine. @reaperhulk should also be added as a contact.